JamJar/AutoCompleteTextField/JamJarCell.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  JamJarCell.swift
+//  JamJar
+//
+//  Created by Ethan Riback on 4/2/16.
+//  Copyright © 2016 JamJar. All rights reserved.
+//
+
+import UIKit
+
+class JamJarCell: UITableViewCell {
+    
+    @IBOutlet weak var thumbnailImageView: UIImageView!
+    @IBOutlet weak var jamjarTitleLabel: UILabel!
+    @IBOutlet weak var viewsLabel: UILabel!
+    @IBOutlet weak var contributorsLabel: UILabel!
+    @IBOutlet weak var lengthTimeLabel: UILabel!
+    
+    var jamjar: JamJarGraph!
+    
+    func setup(jamjar: JamJarGraph, startVideo: Video) {
+        self.jamjar = jamjar
+        
+        // Assign JamJar title
+        jamjarTitleLabel.text = "JamJar Title"
+        
+        // Assign view count
+        viewsLabel.text = "\(123) views"
```

Comment:
In all reals though...not sure if we're gonna have something like a viewcount for this.  Either way not bad to have for now.